logfile

```diff
@@ -192,3 +192,53 @@ STATEMENT:  CREATE DATABASE "daweidai_development" ENCODING = 'unicode'
 ERROR:  database "daweidai_development" is being accessed by other users
 DETAIL:  There is 1 other session using the database.
 STATEMENT:  DROP DATABASE IF EXISTS "daweidai_development"
+LOG:  database system was shut down at 2015-10-13 09:11:44 PDT
```

Comment:
I already have this line in my gitignore file

# Ignore all logfiles and tempfiles.
/log/*.log
logfile
/tmp

Do you know why it is still there?